index.md

```diff
@@ -7,13 +7,13 @@ Good data organization is the foundation of any research project. It not only se
 
 Organizing a project that includes sequencing involves many components. There's the experimental setup and conditions metadata, measurements of experimental parameters, sequencing preparation and sample information, the sequences themselves and the files and workflow of any bioinformatics analysis. So much of the information of a sequencing project is digital, and we need to keep track of our digital records in the same way we have a lab notebook and sample freezer. In this lesson, we'll go through the project organization and documentation that will make an efficient bioinformatics workflow possible. Not only will this make you a more effective bioinformatics researcher, it also prepares your data and project for publication, as grant agencies and publishers increasingly require this information.
 
-In this lesson we'll be using data from a study of experimental evolution using *E. coli*. [More about this dataset](http://www.datacarpentry.org/organization-genomics/data/). In this study there are several types of files
+In this lesson, we'll be using data from a study of experimental evolution using *E. coli*. More information about this dataset is available [here](http://www.datacarpentry.org/organization-genomics/data/). In this study there are several types of files:
```

Comment:
My only suggested change is to hyperlink more than only "here" for accessibility purposes. Something like "[More information about this dataset is available here.](http://www.datacarpentry.org/organization-genomics/data/)". I realize that linking the whole sentence seems like a lot, but it does provide more context.

Reply:
Agreed.